services/checkmk.md

```diff
@@ -0,0 +1,24 @@
+# CheckMK
+
+[CheckMK docs](https://docs.checkmk.com/latest/en/). CheckMK is a tool we use for basic monitoring, including uptime, memory/CPU/disk usage, and more
+
+## Useful commands
+On the host: 
+* Run `sudo cmk-agent-ctl status` to check the agent status on the host.
+* Run `sudo cmk-agent-ctl dump > cmk-dump.txt`
+to verify that the agent is running successfully and see its parameters.
+
+
+
+
+On the CheckMK server:
+* Switch to the ‘pulmonitor’ user (`sudo su - pulmonitor` to run in the new user's environment), then execute `cmk --debug -vvn hostname` to look at the connection to a specific host. 
+* Do `nc -vz hostname.princeton.edu 6556` (probably needs `sudo`) to confirm that the agent port is accessible.
```

Comment:
I got to this step and, even though was sudo'ed into the pulmonitor user, was prompted for a password. Same for accessing the logs below. This may be user error but wanted to note it here in case it's not.

Reply:
Thanks. I just tried this out and found the source of the error/confusion. Suggested update to the PR below.

Reply:
Addressed in cb0ae0040725f92b349176130737fcab410f6fb7.